### PR TITLE
goal pages

### PR DESCRIPTION
**Pull request by vykster**
@corwinstephen started goal pages. need to work on CSS. I think these should all actually just be a layout, with all of the content being generated from a database.

### CODE REVIEW

**Comment by vykster** [comment]
I also need the goal assets that megan designed for white backgrounds.

File: db/cms_fixtures/la2050/pages/home/goals/connect/content.haml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+        %p.prose.summary
+          In 2050, Los Angeles will be the most civically engaged region in the U.S., with multiple online and offline platforms for residents to voice their opinions and influence elected officials, decision-makers, and local government. Our region’s voting rates and volunteerism will lead the nation, and an expanded network of transportation options will facilitate the easy flow of people and ideas.
+
+.background-grey.paddedsection
+  .container
+    .row
+      .col-sm-8.col-sm-offset-2.text-center
+        %h1 Metrics
+        %a{ :href => "/metrics" }
```
Comment (corwinstephen) [inline db/cms_fixtures/la2050/pages/home/goals/connect/content.haml]: You should wrap the `a` in a `p`, not the other way around.

File: db/cms_fixtures/la2050/pages/home/goals/live/content.haml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+          Our region’s residents have the economic means and cultural capital to lead active, healthy lives, and everyone benefits from a sustainable environment.
+        %p.prose.summary
+          In 2050, Los Angeles will have the nation’s lowest obesity rates and every neighborhood will have access to healthy, affordable food. Every family will be able to afford quality health care and housing. And no families will face environmental health hazards because of where they live or how much money they make.
+
+.background-grey.paddedsection
+  .container
+    .row
+      .col-sm-8.col-sm-offset-2.text-center
+        %h1 Metrics
+        %a{ :href => "/metrics" }
```
Comment (corwinstephen) [inline db/cms_fixtures/la2050/pages/home/goals/live/content.haml]: Same thing here

File: db/cms_fixtures/la2050/pages/home/goals/play/content.haml [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+        %p.prose.summary
+          In 2050, people of all ages will enjoy safe homes, safe neighborhoods, and safe places to play. Every neighborhood will provide its residents with vibrant park space, every child will have access to afterschool enrichment programs, and residents will love their neighborhoods.
+
+.background-grey.paddedsection
+  .container
+    .row
+      .col-sm-8.col-sm-offset-2.text-center
+        %h1 Metrics
+        %a{ :href => "/metrics" }
```
Comment (corwinstephen) [inline db/cms_fixtures/la2050/pages/home/goals/play/content.haml]: And here

**Pull request merged**